Dockerfile

```diff
@@ -13,11 +13,27 @@ RUN set -x \
 # Dapper/Drone/CI environment
 FROM build AS dapper
 ENV DAPPER_ENV GODEBUG REPO TAG DRONE_TAG PAT_USERNAME PAT_TOKEN KUBERNETES_VERSION DOCKER_BUILDKIT
+ARG DAPPER_HOST_ARCH
+ENV ARCH $DAPPER_HOST_ARCH
+ENV DRONE_BUILD_EVENT IMAGE_NAME GCLOUD_AUTH
```

Comment:
@erikwilson is is possible that the gcloud bit is not working because of the GCLOUD_AUTH not being in the DAPPER_ENV line above?

Reply:
Thanks! Looks like that fixed it! :)